resource/css/skosmos.css

```diff
@@ -968,12 +973,11 @@ body {
   #vocabulary-list .border-top {
     border-top: 2px var(--dark-color) solid !important;
     color: var(--secondary-dark-color);
-    text-decoration: underline var(--secondary-medium-color) solid;
   }
```

Comment:
Everything from line 963 to 976 should be moved to line 400 with the rest of the vocab list definitions.